website/docs/docs/dbt-versions/release-notes/04-Nov-2022/ide-features-ide-deprecation.md

```diff
@@ -0,0 +1,32 @@
+---
+
+title: "Extra features in new IDE, and classic IDE deprecation"
+id: "ide-features-ide-deprecation"
+description: "Enhancement and Deprecation: Extra features in new IDE, and classic IDE deprecation"
+sidebar_label: "Enhancement and Deprecation: Extra features in new IDE, and classic IDE deprecation"
+tags: [Nov-29-2022]
+
+---
+
+### Extra features in new and refreshed IDE
+
+The new and refreshed version of the Cloud IDE has launched four brand-new additional features, making it easier and faster for you to develop in the IDE.
+
+The new features are:
+
+- **Formatting** &mdash; Format your files with a click of a button, powered by sqlfmt.
+- **Git diff view** &mdash; Ability to see what you've changed in a file before you make a pull request.
+- **dbt autocomplete** &mdash; There are four new types of autocomplete features to help you develop faster:
+    - Use ref to autocomplete your model names
+    - Use source to autocomplete your source name + table name
+    - Use macro to autocomplete your arguments
+    - Use env var to autocomplete env var
+- **Dark mode**	&mdash;  Use dark mode in the Cloud IDE for a great viewing experience in low-light environments.
+
+Read more about all the [Cloud IDE features](/docs/get-started/dbt-cloud-features).
+
+### Classic IDE deprecation notice
+
+In December 2022, dbt Labs will deprecate the classic IDE in favor of the [new and refreshed IDE](/docs/get-started/develop-in-the-cloud) for _all_ dbt Cloud users. You will no longer be able to access the classic IDE and dbt Labs may introduce changes that break the classic IDE.
+
+The dbt Labs Support team will only support and resolve any issues related to the new and refreshed IDE. 
```

Comment:
```suggestion
With deprecation, only the new and refreshed IDE will be supported. 
```

Reply:
sugg: conciseness

Reply:
updated my Add a suggestion, per our meeting, @mirnawong1 ! feel free to commit it when you have a chance!